### PR TITLE
Skip empty rows during CSV import, refs #11646

### PR DESCRIPTION
**Pull request by mcantelon**
During CSV import, check each row column to make sure that there is at least
one non-blank value before importing the row.

### CODE REVIEW

File: lib/QubitFlatfileImport.class.php [hunk 1]
```diff
@@ -611,6 +611,13 @@ public function csv($fh, $skipRows = 0)
     {
       if ($this->status['rows'] >= $skipRows)
       {
+        // skip blank rows, but keep track of rows parsed
```
Comment (jraddaoui) [inline lib/QubitFlatfileImport.class.php]: Uppercase

File: lib/QubitFlatfileImport.class.php [hunk 2]
```diff
@@ -653,6 +660,24 @@ public function csv($fh, $skipRows = 0)
     $this->executeClosurePropertyIfSet('completeLogic');
   }
 
+  /**
+   * Filter out elements containing blank strings and check if any elements
+   * remain
+   *
+   * @param array $row  array of column values
```
Comment (jraddaoui) [inline lib/QubitFlatfileImport.class.php]: Uppercase in the second array

**Review by jraddaoui — APPROVED** [review]
Looks good! Just a couple of cosmetic changes.

**Comment by sbreker** [comment]
Looks good.  See Radda's comments above.

**Pull request merged**